tests/bson/bson-ods-002.phpt

```diff
@@ -1,131 +1,145 @@
 --TEST--
 BSON encoding: Encoding data into BSON representation, and BSON into Extended JSON
+--XFAIL--
+Persistable::bsonSerialize() should not return atomic modifiers (see: PHPC-550)
 --INI--
 date.timezone=America/Los_Angeles
 --SKIPIF--
-<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; CLEANUP(STANDALONE); ?>
 --FILE--
 <?php
-use MongoDB\BSON as BSON;
-
 require_once __DIR__ . "/../utils/basic.inc";
 
-class Address implements BSON\Persistable {
-    protected $streetAddress;
-    protected $city;
-    protected $postalCode;
+use MongoDB\BSON as BSON;
```

Comment:
Please don't change test cases! I realize it no longer works, and hence the XFAIL is fine, but I don't see why the test itself should be so massively changed.